TradingBot/Services/Strategy/GreedyStrategy.cs

```diff
@@ -65,6 +65,11 @@ public async Task<IReadOnlyList<TradingAction>> GetTradingActionsAsync(Cancellat
 
         var predictions = await _predictor.GetPredictionsAsync(token);
         var assets = await _assetsDataSource.GetCurrentAssetsAsync(token);
+        var predictionsForHeldTokens = await _predictor.GetPredictionsForListOfSymbolsAsync(assets.Positions.Keys.ToList(), token);
```

Comment:
It would be better to include held tokens in `GetPricesAsync()`, since that's what happens in normal execution. I fixed this on `visualisations-and-tests` branch, so you can just cherry-pick it.